packages/common/src/permissions/HubPermissionPolicies.ts

```diff
@@ -108,6 +108,18 @@ const SystemPermissionPolicies: IPermissionPolicy[] = [
     // we pass `?pe=hub:feature:user:preferences` in the URL
     environments: ["devext", "qaext"],
   },
+  {
+    permission: "hub:environment:qaext",
+    environments: ["qaext"],
+  },
+  {
+    permission: "hub:environment:devext",
+    environments: ["devext"],
+  },
+  {
+    permission: "hub:environment:production",
+    environments: ["production"],
+  },
```

Comment:
I'd maybe add a quick comment here saying to use these only if necessary -- we'd rather people make permissions where they specify the environment, but if that can't happen (i.e. in the case of notices where there isn't an entity type to write permissions for), then we can utilize these.

Reply:
I agree with @jordantsanz - these should be limited use, and we definitely don't want to have them used in the dependencies arrays. But I do like using permissions >> `context.environment === "qaext"` sort of stuff

Reply:
Added :)